torch/quantization/quantize.py

```diff
@@ -247,7 +258,7 @@ def swap_module(mod, mapping):
         if type(mod) in mapping:
             new_mod = mapping[type(mod)].from_float(mod)
 
-    if type(mod) == DeQuantStub:
+    if swap_dequant and type(mod) == DeQuantStub:
```

Comment:
this is sketchy - maybe instead just say `if DequantStub in mapping`? (so the only difference is that we ignore the observer field)

Reply:
I can add qconfig to DeQuantStub and remove this argument

Reply:
why this is still here? I think we don't have the most recent code..